### DISCUSSION TITLE
Not started?

### DISCUSSION THREAD

**Discussion by jongfeel** [humansof42seoul/Humansof42-v2#8] [Q&A] [answered]
궁금해서 그런데 아직 시작을 안한건가요?
아니면 시작 하셨는데 아직 github에는 내용이 없는 건가요?
지금 상황/상태를 알고 싶습니다.
(화난거 아님)

**Comment by jwon42** [comment, marked as answer]
지체됐습니다. 속도 올리겠습니다.
